industrial_taxonomy/getters/glass_house.py

```diff
@@ -0,0 +1,60 @@
+"""Data getter for Glass to Companies House matching."""
+from functools import lru_cache
+from typing import Dict, List, Optional, TypedDict
+
+from metaflow import Flow, Run
+from metaflow.exception import MetaflowNotFound
+
+from industrial_taxonomy import config
+
+
+if config is None:
+    raise FileNotFoundError("Could not find config file.")
+MATCH_THRESHOLD: int = config["glass_house"]["match_threshold"]
+
+# Type aliases
+glass_id = int
+company_number = str
+
+
+class FullMatchResult(TypedDict):
```

Comment:
Add a doc-string to explain what this does?

Reply:
I'm not sure what would you like to know.

It might be a misunderstanding of what it's doing/for...
It in itself is specifying a type that is the return type of  `glass_companies_house` matches. It's specifying that the type `FullMatchResult` is a dictionary with keys `sim_mean`, `org_id`, `org_name`, `company_number`, `company_name` (and the corresponding types).

More info here: https://www.python.org/dev/peps/pep-0589/

Reply:
Thanks for clarifying!